vm/ops/groups.cpp

```diff
@@ -419,6 +419,69 @@ void x86CPU::op_group_D3(){
 	}
 }
 
+void x86CPU::op_group_C0(){
+    ModRM rm(this);
+    switch(rm.GetExtra()){
+        case 7:
+		op_sar_rm8_imm8(rm);
+		break;
+		case 4:
+		op_shl_rm8_imm8(rm);
+		break;
+		case 5:
+		op_shr_rm8_imm8(rm);
+		break;
+		case 0:
+		op_rol_rm8_imm8(rm);
+		break;
+		case 1:
+		op_ror_rm8_imm8(rm);
+		break;
+		case 3:
+		op_rcr_rm8_imm8(rm);
+		break;
+		case 2:
+		op_rcl_rm8_imm8(rm);
```

Comment:
bug here and at the same place in C1. There is no break, so when you execute this opcode it would fall down into the unknown opcode exception

Reply:
oops, a stupid bug. And I have fixed it. Thank you.